src/lib/tree/Constants.sol

```diff
@@ -16,5 +16,6 @@ library Constants {
     bytes1 internal constant NODE_PREFIX = 0x01;
 
     /// @dev Parity share namespace ID
-    NamespaceID internal constant PARITY_SHARE_NAMESPACE_ID = NamespaceID.wrap(0xFFFFFFFFFFFFFFFF);
+    NamespaceID internal constant PARITY_SHARE_NAMESPACE_ID =
+        NamespaceID.wrap(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF);
```

Comment:
[optional] in a separate PR it may make sense to rename namespace ID to namespace for similar reasons to https://github.com/celestiaorg/nmt/issues/206

Reply:
https://github.com/celestiaorg/quantum-gravity-bridge/issues/172